styles/colors-cobalt.less

```
@@ -153,7 +155,7 @@
 @agate-picker-item-decrement-bg-image: none;
 @agate-picker-item-increment-bg-image: none;
 @agate-picker-item-focus-color: @agate-foreground-focus;
-@agate-picker-item-focus-bg-color: @agate-background-focus;
+@agate-picker-item-focus-bg-color: @agate-focus-color;
```

Could you please check it? Thank you.
```suggestion
@agate-picker-item-focus-bg-color: @agate-focus-bg-color;
```